cmd/geth/dbcmd.go

```diff
@@ -407,7 +407,7 @@ func checkStateContent(ctx *cli.Context) error {
 	return nil
 }
 
-func showLeveldbStats(db ethdb.KeyValueStater) {
```

Comment:
It doesn't make sense to only change the function name, as this function is apparently bound to LevelDB.

Instead, I would prefer to improve this function to also support Pebble.

Reply:
Yes! By the way, I think that it still query and show all db metrics from Pebble already, not only bound to LevelDB!